.github/workflows/ci.yaml

```diff
@@ -39,6 +39,36 @@ jobs:
     - run: make dev
     - run: make test
 
+  integration-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        CONNECT_VERSION:
+          - 2024.05.0
+          - 2024.04.1
+          - 2024.04.0
+          - 2024.03.0
+          - 2024.02.0
+          - 2024.01.0
+          - 2023.12.0
+          - 2023.10.0
+          - 2023.09.0
+          - 2023.07.0
+          - 2023.06.0
+          - 2023.05.0
+          - 2023.01.1
+          - 2023.01.0
+          - 2022.12.0
+          - 2022.11.0
+    steps:
+    - uses: actions/checkout@v4
+    - uses: docker/setup-buildx-action@v3
+    - uses: docker/setup-qemu-action@v3
```

Comment:
What do we need qemu for? Everything in this stack is x86 right?

Reply:
Oh, good catch. I'll try removing it.